modules/passkey/test/4337/Safe4337Module.spec.ts

```diff
@@ -58,10 +60,10 @@ describe('Safe4337Module', () => {
     }
   })
 
-  describe('SafeSignerLaunchpad', () => {
+  describe('SafeWebAuthnSigner', () => {
```

Comment:
These are just the tests moved from `modules/passkey/test/4337/SafeWebAuthnSigner.spec.ts`